packages/@uppy/dashboard/src/components/Dashboard.js

```diff
@@ -81,6 +81,7 @@ module.exports = function Dashboard (props) {
       onDrop={props.handleDrop}
     >
       <div
+        aria-hidden="true"
```

Comment:
Why was this added?

Reply:
```
   83:7   error    Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events
   83:7   error    Static HTML elements with event handlers require a role                                         jsx-a11y/no-static-element-interactions
```
Since it's the overlay, I thought the easiest thing to do was to hide it from screen readers, but maybe there's a better way?